crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -18,14 +18,58 @@ pub(super) use frozen::FrozenSlidingSyncList;
 use futures_core::Stream;
 pub(super) use request_generator::*;
 pub use room_list_entry::RoomListEntry;
-use ruma::{api::client::sync::sync_events::v4, assign, events::StateEventType, OwnedRoomId};
+use ruma::{
+    api::client::sync::sync_events::v4, assign, events::StateEventType, OwnedRoomId, TransactionId,
+};
 use serde::{Deserialize, Serialize};
 use tokio::sync::mpsc::Sender;
 use tracing::{instrument, warn};
 
-use super::{Error, SlidingSyncInternalMessage};
+use super::{
+    sticky_parameters::{StickyData, StickyManager},
+    Error, SlidingSyncInternalMessage,
+};
 use crate::Result;
 
+#[derive(Debug)]
+pub(super) struct ListStickyParameters {
```

Comment:
```suggestion
pub(super) struct SlidingSyncListStickyParameters {
```

to be consistent with the other namings.